kv/dist_sender.go

```diff
@@ -48,7 +48,7 @@ const (
 	// from an internal range lookup.
 	defaultRangeLookupMaxRanges = 8
 	// The default size of the leader cache.
-	defaultLeaderCacheSize = 1 << 20
+	defaultLeaderCacheSize = 1 << 6
```

Comment:
should probably come up with a sensible default. `1<<20` wasn't more than a couple of megs of ram. `1<<6` seems really small. @spencerkimball @bdarnell